### ISSUE TITLE
test-functional-oauth grep="oauth reset password" fails locally

### ISSUE THREAD

**Issue by vladikoff** [mozilla/fxa-content-server#5725]
```
npm run test-functional-oauth grep="oauth reset password"
```

The above fails for me locally with an "unexpected error"

@vbudhram @philbooth  @shane-tomlinson could one of you run that test on your machine please?

**Comment by vbudhram** [comment]
@vladikoff This failed for me too. Noticed that 123Done uses node 4 and wasn't starting properly in my environment. After starting it in node 4 it still errored out with

```
× firefox on any platform - oauth reset password - reset password, verify same browser (0.179s)
UnknownError: [POST http://localhost:4444/wd/hub/session/518efa41-0c58-9548-83cb-f7b67be4f747/element/86bce3bf-953d-b349-8b6f-59b0b14769b9/click / {}] Reached error page: about:neterror?e=dnsNotFound&u=https%3A//oauth.dev.lcip.org/v1/authorization%3Fclient_id%3Ddcdb5ae7add825d2%26state%3D24a2ff52a1ff4a97ed9f4a189964a5c3020ce31605fe553e0214576819198757%26scope%3Dprofile%26redirect_uri%3Dhttps%253A%252F%252F123done.dev.lcip.org%252Fapi%252Foauth%26action%3Dsignin&c=UTF-8&f=regular&d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20oauth.dev.lcip.org.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:40.131Z'
System info: host: 'vbudhram-23693', ip: '192.168.0.12', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '1.8.0_131'
Driver info: driver.version: unknown
  at Command.<anonymous>  <tests/functional/lib/helpers.js:188:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:56:25>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:1282:8>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:56:25>
  at Test.reset password, verify same browser [as test]  <tests/functional/oauth_reset_password.js:54:10>
```

**Comment by vbudhram** [comment]
Circled back to this and after updating 123Done (and using node 6) I get all passing tests! FWIW, I am on v6.11.5.

![screen shot 2017-11-17 at 9 33 14 am](https://user-images.githubusercontent.com/1295288/32952165-5fb4049a-cb7a-11e7-9a9f-c8b45ca57ef5.png)

**Comment by shane-tomlinson** [comment]
I get all passing tests too:

```bash
✓ firefox on any platform - oauth reset password - reset password, verify same browser (4.599s)
✓ firefox on any platform - oauth reset password - reset password, verify same browser with original tab closed (4.454s)
✓ firefox on any platform - oauth reset password - reset password, verify same browser by replacing the original tab (3.851s)
✓ firefox on any platform - oauth reset password - reset password, verify in a different browser, from the original tab's P.O.V. (6.854s)
✓ firefox on any platform - oauth reset password - reset password, verify in a different browser, from the new browser's P.O.V. (3.582s)
```

@vladikoff - which version of Firefox are you pointing at? I tested with both 56.0.1 and 57, using both headless and non-headless.

**Comment by vladikoff** [comment]
The error I get

````
2|auth-ser | fxa-auth-server.ERROR: request.summary {"op":"request.summary","code":500,"errno":999,"rid":"1511390044205:vladikoffs-MBP:24019:jabm0gzn:10063","path":"/v1/account/reset","lang":"en-US,en;q=0.9,pt;q=0.8,af;q=0.7","agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36","remoteAddressChain":["127.0.0.1"],"t":1439,"uid":"2df738052fa24dc7b04d29808f35518e","stack":"Error: connect ECONNREFUSED 127.0.0.1:11211\n    at Object.exports._errnoException (util.js:1020:11)\n    at exports._exceptionWithHostPort (util.js:1043:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)"}
```

I guess y'all got that running, but I didn't have that. Another reason for https://github.com/mozilla/fxa-local-dev/issues/94 to happen.
